core-bundle/src/Resources/contao/library/Contao/DcaLoader.php

```diff
@@ -114,6 +116,59 @@ public function load($blnNoCache=false)
 			include $rootDir . '/system/config/dcaconfig.php';
 		}
 	}
+
+	/**
+	 * Adds the default labels (see #509)
+	 */
+	private function addDefaultLabels()
+	{
+		// Operations
+		foreach (array('global_operations', 'operations') as $key)
+		{
+			if (!isset($GLOBALS['TL_DCA'][$this->strTable]['list'][$key]))
+			{
+				continue;
+			}
+
+			foreach ($GLOBALS['TL_DCA'][$this->strTable]['list'][$key] as $k=>&$v)
+			{
+				if (isset($v['label']))
+				{
+					continue;
+				}
+
+				if (isset($GLOBALS['TL_LANG'][$this->strTable][$k]))
+				{
+					$v['label'] = $GLOBALS['TL_LANG'][$this->strTable][$k];
```

Comment:
Is it on purpose that you didn’t use references to the `$GLOBALS['TL_LANG']` array here?

Reply:
No. I have to fix this.

Reply:
Fixed in c8b6e628552a3e1520614ca743c93485f13a71aa.